os/arch/arm/src/armv7-a/arm_mmu.c

```diff
@@ -508,27 +512,27 @@ void mmu_dump_pgtbl(void)
 
 	uint32_t *l1tbl = mmu_l1_pgtable();
 
-	lldbg("L1 page table base addr = 0x%08x\n", l1tbl);
+	lldbg_noarg("L1 page table base addr = 0x%08x\n", l1tbl);
 
-	lldbg("=========================================\n");
-	lldbg("ENTRY      TYPE    OUT             ACCESS\n");
-	lldbg("ADDR               ADDR                  \n");
-	lldbg("=========================================\n");
+	lldbg_noarg("=====================================================================\n");
```

Comment:
Will you handle this for the TRAP later? @thapav

Reply:
Yes Chang.. We will make sure to raise a separate PR for TRAP.